articles/openshift/oshift-how-expand-persistent-vols.md

```diff
@@ -0,0 +1,162 @@
+---
+title: How to expand OpenShift persistent volumes | UKCloud Ltd
+description: Outlines methods to expand OpenShift persistent volumes in v3.11 OpenShift clusters.
+services: openshift
+author: Gareth Ellner
+reviewer:
+lastreviewed: 
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to expand OpenShift persistent volumes
+toc_fullpath: How To/oshift-how-expand-persistent-vols.md
+toc_mdlink: oshift-how-expand-persistent-vols.md
+---
+
+# How to expand OpenShift persistent volumes
+
+## Overview
+
+In OpenShift clusters running OpenShift v3.11 or later, it is possible to expand existing persistent volumes. It is necessary to shutdown any pod/container using the volume before the expansion can be performed. In earlier versions (such as v3.9) it is necessary to raise a support request so that UKCloud admins can facilitate the expansion.
```

Comment:
```suggestion
In OpenShift clusters running OpenShift v3.11 or later, it is possible to expand existing persistent volumes. It is necessary to scale-down any pod/container using the volume before the expansion can be performed. In earlier versions (such as v3.9) it is necessary to raise a support request so that UKCloud admins can facilitate the expansion.
```